src/EditModel/SectionDialog.component.js

```diff
@@ -359,6 +374,15 @@ class SectionDialog extends React.Component {
         });
     };
 
+    handleSectionTextUpdate = (fieldName) => (ev, text) => {
+        this.setState({
```

Comment:
To be pedantic this should use the callback-style for prevValue, right?
Shouldn't really matter, though...